inventory-fusion/src/test/scala/com/normation/inventory/provisioning/fusion/TestReportParsing.scala

```diff
@@ -154,6 +151,11 @@ class TestReportParsing extends Specification with Loggable {
       agents must be empty
     }
 
+    "should have dsc agent agent when using rudder-agent based on dsc" in {
+      val agents = parser.parse("fusion-report/dsc-agent.ocs").node.agents.map(_.name).toList
+      agents == (DSC_AGENT :: Nil)
+    }
+
```

Comment:
Please also add a test for the agent version below (in "Agent version" should .... )